pkg/util/git.go

```diff
@@ -215,61 +198,51 @@ func parseBitbucketUrl(g GitUrl, url *url.URL, c http.Client) (GitUrl, error) {
 		}
 	}
 
-	if !isGitUrlPublic(g, c) {
-		g.token = os.Getenv(BitbucketToken)
-	}
-
-	return g, err
+	return err
 }
 
-func isGitUrlPublic(g GitUrl, c http.Client) bool {
-	host := g.Host
-	if host == RawGitHubHost {
-		host = GitHubHost
-	}
-
-	repo := fmt.Sprintf("%s://%s/%s/%s", g.Protocol, host, g.Owner, g.Repo)
-
-	if res, err := c.Get(repo); err != nil {
-		return false
-	} else if res.StatusCode == http.StatusOK {
-		return true
-	}
-	return false
-}
-
-// CloneGitRepo clones a GitHub Repo to a destination directory
+// CloneGitRepo clones a git repo to a destination directory
```

Comment:
Describe the expected format for `destDir` and ensure it gets validated.   If invalid, issue a user error